src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -374,6 +391,9 @@ export class EditDataGridPanel extends GridParentComponent {
 				return Promise.resolve();
 			},
 			(error) => {
+				self.telemetryService.createActionEvent(TelemetryKeys.TelemetryView.EditDataGrid, TelemetryKeys.TelemetryError.EditCellSelectError)
+					.withAdditionalProperties({ error: error })
```

Comment:
Generally we can't send the full error message since that can contain user data (such as data values). If you can't guarantee that the message is safe to send then you'll need to either remove this or search for certain well-known error stings you expect and send a hardcoded value instead (such as NullRefError, UserCancelled, Unknown, etc.)